src/category_theory/abelian/opposite.lean

```diff
@@ -29,4 +31,52 @@ instance : abelian Cᵒᵖ :=
   normal_epi := λ X Y f m, by exactI
     normal_epi_of_normal_mono_unop _ (abelian.normal_mono f.unop), }
 
+section
+
+variables {X Y : C} (f : X ⟶ Y)
+
+-- TODO: Generalize (this will work whenever f has a cokernel)
+-- (The abelian case is probably sufficient for most applications.)
+/-- The kernel of `f.op` is the opposite of `cokernel f`. -/
+@[simps]
+def kernel_op_unop : (kernel f.op).unop ≅ cokernel f :=
```

Comment:
There are a bunch of variants of this, eg `kernel f.op ≅ (cokernel f).op` or `(kernel f).unop ≅ cokernel f.unop`. Do you think it's worth explicitly adding any of the others? I guess the latter one I mention is defeq to an application of `kernel_op_unop`...

Reply:
I think these can all be obtained with combinations of the two I made and `iso.(un)op`. But it's probably worthwhile to add these anyway...

Reply:
I'm happy either way, if you're convinced they're all easy modifications of the existing ones!

Reply:
Okay, I added a few variants.